src/RecoveryServices/RecoveryServices.Backup.Helpers/TrackingHelpers.cs

```diff
@@ -110,6 +110,34 @@ public class TrackingHelpers
             return opStatusResponse;
         }
 
+        /// <summary>
+        /// Block to track the operation to completion.
+        /// Waits till the HTTP status code of the operation becomes something other than Accepted.
+        /// </summary>
+        /// <param name="response">Response of the operation returned by the service.</param>
+        /// <param name="getOpStatus">Delegate method to fetch the operation status of the operation.</param>
+        /// <returns>Result of the operation once it completes.</returns>
+        public static RestAzureNS.AzureOperationResponse<T> GetOperationResult<T>(
+            RestAzureNS.AzureOperationResponse response,
+            Func<string, RestAzureNS.AzureOperationResponse<T>> getOpStatus)
+            where T: ServiceClientModel.ProtectionContainerResource
+        {
+            var operationId = response.Response.Headers.GetOperationResultId();
+
+            var opStatusResponse = getOpStatus(operationId);
+
+            while (opStatusResponse.Response.StatusCode == SystemNet.HttpStatusCode.Accepted)
+            {
+                TestMockSupport.Delay(_defaultSleepForOperationTracking * 1000);
```

Comment:
@siddharth7 this is fine since it's already being used in this file, but in general, we should avoid using test code in assemblies that we are shipping.

Reply:
I will look into this portion sometime, this is some old code that I am not aware about